lein-jank/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -xe
```

Comment:
`set -euo pipefail` is a good default. Do we need `-x` here?

Reply:
We don't need -x.